package.json

```diff
@@ -41,17 +35,17 @@
     "classnames": "^2.2.6",
     "d3-dsv": "^1.1.1",
     "d3-scale-chromatic": "^1.3.3",
-    "deck.gl": "^8.1.0-alpha.1",
+    "deck.gl": "8.1.0-alpha.1",
     "json2csv": "^4.5.2",
     "math.gl": "^3.1.3",
     "nebula.gl": "^0.17.1",
-    "openseadragon": "^2.4.0",
+    "node-sass": "^4.13.1",
     "prop-types": "^15.7.2",
     "pubsub-js": "^1.7.0",
-    "rc-tree": "^2.1.0",
-    "react": "^16.8.6",
+    "rc-tree": "2.1.0",
+    "react": "16.8.6",
     "react-color": "^2.18.0",
-    "react-dom": "^16.8.6",
+    "react-dom": "16.8.6",
```

Comment:
Why are the `^` pins going away? For myself, I'd prefer if that weren't the default in NPM, but it is, so we should have a reason for departing from it.

Reply:
The newer version of react caused eslint to throw this error:

```
error  componentWillMount is deprecated since React 16.9.0, use UNSAFE_componentWillMount instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillmount. Use https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles to automatically update your components  react/no-deprecated
```

Reply:
Do we want to upgrade our react version and switch to using the UNSAFE_ prefix? If so I can work on that